.github/actions/setup-sentry/action.yml

```diff
@@ -0,0 +1,143 @@
+  pip-cache-dir:
+    description: "Path to pip cache"
+    value: ${{ steps.config.outputs.pip-cache }}
+  acceptance-dir:
+    description: "Path to acceptance visual snapshot artifacts"
+    value: ${{ steps.config.outputs.acceptance-dir }}
+  matrix-instance-number:
+    description: "The matrix instance number (starting at 1)"
+    value: ${{ steps.config.outputs.matrix-instance-number }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup default environment variables
+      shell: bash
+      env:
+        NEED_KAFKA: ${{ inputs.kafka }}
+        MATRIX_INSTANCE: ${{ matrix.instance }}
+        MATRIX_INSTANCE_TOTAL: ${{ strategy.job-total }}
+      run: |
+        echo "PIP_DISABLE_PIP_VERSION_CHECK=on" >> $GITHUB_ENV
+        echo "MIGRATIONS_TEST_MIGRATE=0" >> $GITHUB_ENV
+        echo "SENTRY_LIGHT_BUILD=1" >> $GITHUB_ENV
+        echo "SENTRY_SKIP_BACKEND_VALIDATION=1" >> $GITHUB_ENV
+
+        ### node configuration ###
+        echo "NODE_ENV=development" >> $GITHUB_ENV
+        echo "NODE_OPTIONS=--max-old-space-size=4096" >> $GITHUB_ENV
+
+        ### pytest-sentry configuration ###
+        echo "PYTEST_SENTRY_DSN=https://6fd5cfea2d4d46b182ad214ac7810508@sentry.io/2423079" >> $GITHUB_ENV
+        echo "PYTEST_ADDOPTS=--reruns 5" >> $GITHUB_ENV
+        # this handles pytest test sharding
+        if [ "$MATRIX_INSTANCE" ]; then
+          echo "TEST_GROUP=$MATRIX_INSTANCE" >> $GITHUB_ENV
+          echo "TOTAL_TEST_GROUPS=$MATRIX_INSTANCE_TOTAL" >> $GITHUB_ENV
+        fi
+        # This records failures on master to sentry in order to detect flakey tests, as it's
+        # expected that people have failing tests on their PRs
+        [ "$GITHUB_REF" = "refs/heads/master" ] && echo "PYTEST_SENTRY_ALWAYS_REPORT=1" >> $GITHUB_ENV || true
+
+        ### services configuration ###
+        echo "BIGTABLE_EMULATOR_HOST=localhost:8086" >> $GITHUB_ENV
+        # Note: some backend tests (e.g. tests/sentry/eventstream/kafka/test_consumer.py) will behave
+        # differently if these are set.
+        if [ "$NEED_KAFKA" = "true" ]; then
+          echo "SENTRY_KAFKA_HOSTS=127.0.0.1:9092" >> $GITHUB_ENV
+          echo "SENTRY_ZOOKEEPER_HOSTS=127.0.0.1:2181" >> $GITHUB_ENV
+        fi
+
+    - name: Install system dependencies
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y --no-install-recommends \
+          libxmlsec1-dev \
+          libmaxminddb-dev
+
+    - name: Set up outputs
+      id: config
+      env:
+        MATRIX_INSTANCE: ${{ matrix.instance }}
+      shell: bash
+      run: |
+        echo "::set-output name=yarn-cache-dir::$(yarn cache dir)"
+        echo "::set-output name=matrix-instance-number::$(($MATRIX_INSTANCE+1))"
+        echo "::set-output name=acceptance-dir::.artifacts/visual-snapshots/acceptance"
+
+    - name: Install python dependencies
+      shell: bash
+      # Modified from upstream to make things work
+      run: |
+        cd sentry
```

Comment:
@billyvg Changing to the right directory is the main reason that I forked the action.

Reply:
Does this (working directory) persist across the rest of the steps?